zanata-war/src/main/java/org/zanata/servlet/KLoginServlet.java

```diff
@@ -0,0 +1,98 @@
+/*
+ * Copyright 2016, Red Hat, Inc. and individual contributors as indicated by the
+ * @author tags. See the copyright.txt file in the distribution for a full
+ * listing of individual contributors.
+ *
+ * This is free software; you can redistribute it and/or modify it under the
+ * terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This software is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this software; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA, or see the FSF
+ * site: http://www.fsf.org.
+ */
+package org.zanata.servlet;
+
+import org.zanata.security.AuthenticationManager;
+import org.zanata.security.UserRedirectBean;
+
+import javax.inject.Inject;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+
+/**
+ * Servlet which serves as a landing place to perform kerberos ticket
+ * based authentication.
+ * @author Carlos Munoz <a href="mailto:camunoz@redhat.com">camunoz@redhat.com</a>
+ */
+@WebServlet(
+        urlPatterns = {"/account/klogin"}
+)
+public class KLoginServlet extends HttpServlet {
+
+    @Inject
+    private UserRedirectBean userRedirect;
+
+    @Inject
+    private AuthenticationManager authenticationManager;
+
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        // Set the continue parameter
+        String continueUrl = req.getParameter("continue");
+        if(continueUrl != null)
+            userRedirect.setEncodedUrl(continueUrl);
```

Comment:
This should have failed checkstyle:  missing space after `if`, missing braces for `if` body.